ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt

```diff
@@ -0,0 +1,42 @@
+import io.ktor.client.features.websocket.*
+import io.ktor.http.cio.websocket.*
+import kotlinx.coroutines.channels.*
+import kotlin.coroutines.*
+import kotlin.test.*
+
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
Should the license info be on top of the file?

Reply:
Sorry, IDEA autoimport. Fixed